gammapy/maps/tests/test_wcs.py

```diff
@@ -199,3 +199,19 @@ def test_geom_repr():
     geom = WcsGeom.create(skydir=(0, 0), npix=(10, 4), binsz=50,
                           coordsys='GAL', proj='AIT')
     assert geom.__class__.__name__ in repr(geom)
+
+def test_region_mask():
+    from regions import CircleSkyRegion
+    geom = WcsGeom.create(npix=(3, 3), binsz=2,
+                          proj='CAR', coordsys='GAL')
+
+    region = CircleSkyRegion(SkyCoord(0, 0, unit='deg', frame='galactic'),
+                             1.0 * u.deg)
+
+    mask = geom.region_mask([region], inside=True)
```

Comment:
Maybe change the one test case to have two regions that partly overlap, like in a Venn diagram?
I guess it doesn't really help to test much?
Does your test case have pixels outside the image?
Suggest to have that, to make sure there's no IndexError or such things.
Could even have a region completely outside, just to show that it has no effect.

Reply:
Good point. I added a region outside the image.